### PR TITLE
Do not check against snapshot base in bnd-baseline-maven-plugin

### PR DESCRIPTION
**Pull request by alien11689**
Full story:
1. I have created an interface with annotation Version("1.0.0") and run mvn clean install
2. The bundle has version 1.0.0-SNAPSHOT
3. I have released it via maven release plugin
4. I have version 1.0.0 and 1.0.0-SNAPSHOT in my local .m2
5. I have built it next time and the bundle with version 1.0.1-SNAPSHOT has been installed in .m2
6. I have changed something incompatible in interface so baseline have told me to bump versions to 2.0.0 and bump major part of bundle version
7. I have done it as described above (package version to 2.0.0 and bundle version to 2.0.0-SNAPSHOT)
8. I have run build and version 1.0.1-SNAPSHOT has been selected by plugin as base version, not 1.0.0 (as it should, as it expected)

Conclusion:
It should be possible to exclude SNAPSHOT bundles as base version in baselining

### CODE REVIEW

**Comment by bjhargrave** [comment]
You need to amend your commits so that each commit message includes a `Signed-off-by` as required by CONTRIBUTING. Then push force your updated commits. Thanks.

**Comment by bjhargrave** [comment]
There is no description of the purpose of the requested change and some justification for why it should be made. Please put a meaning description in the commit comment and also this PR. Thanks.

**Comment by alien11689** [comment]
@bjhargrave I have added `Signed-off-by` header and description to commit

File: maven/bnd-baseline-maven-plugin/src/main/java/aQute/bnd/maven/baseline/plugin/BaselineMojo.java [hunk 1]
```diff
@@ -61,6 +62,9 @@
 	@Parameter(property = "bnd.baseline.continue.on.error", defaultValue = "false")
 	private boolean					continueOnError;
 
+	@Parameter(property = "bnd.baseline.compare.to.snapshot", defaultValue = "true")
+	private boolean					compareToShanpshot;
```
Comment (bjhargrave) [inline maven/bnd-baseline-maven-plugin/src/main/java/aQute/bnd/maven/baseline/plugin/BaselineMojo.java]: This variable name is misspelled.

**Pull request closed without merging**